schema/elasticsearch/v6/visibility/index_template.json

```diff
@@ -83,7 +83,8 @@
             "Operator": { "type": "keyword"},
             "RolloutID": { "type": "keyword"},
             "BinaryChecksums": { "type": "keyword"},
-            "Passed": { "type": "boolean" }
+            "Passed": { "type": "boolean" },
+            "CronSchedule": { "type": "keyword" }
```

Comment:
What is the difference between adding it here and adding it outside Attr property?

Reply:
I was thinking not all workflows need this field, but it should be ok since the field will not be present in visibility store if we don't send them. And we will no longer need to worry about if oss user has used them in search attribute